src/App.tsx

```diff
@@ -103,17 +109,25 @@ export default function App() {
             />
           </Grid>
         </Grid>
-        <Grid container justify="flex-end">
+
+        <Grid container justify="space-between">
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => {
+              setMeasureFileName(null);
+              setPatientFileName(null);
+              setMeasureBundle(null);
+              setPatientBundle(null);
+            }}
```

Comment:
Can we also reset the calculation options and measurement period? You can reset the calculation options such that everything is false, and the measurement period back to 1/1/19-12/31/19